benchmarks/pipe.py

```diff
@@ -566,19 +476,18 @@ def run_mp_worker(args, available_workers):
         input_device=torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu"),
         pipelined_backward=args.pipelined_backward,
         checkpoint=args.checkpoint,
-        # loss_fn=blob["criterion"],
+        # FIXME: Do we need to comment this out? loss_fn=benchmark_config["criterion"],
```

Comment:
it seems that the test need to have an option to test either this line is commented out or not?

Reply:
I am not sure at the moment. I am going to tackle this when I enable the benchmark again.